templates/container/template/model.xml

```diff
@@ -26,6 +26,9 @@
         <tags>
             <tag type="regular">layout</tag>
             <tag type="section">layouts</tag>
+            <% _.each(tags, function(tag){ %>
+            <tag type="<%= tag%>"></tag>
```

Comment:
Move the <%= tag %> as the value of the tag and have the type as 'regular' by default.